indico/modules/events/__init__.py

```diff
@@ -231,3 +232,9 @@ def _category_moved(category, old_parent, new_parent, **kwargs):
         assert int(event.as_legacy.getOwner().id) == int(category.id)
         # this will also update the chain (sqlalchemy hook)
         event.category_id = int(category.id)
+
+
+@template_hook('event-references-list')
+def _inject_event_references(event, **kwargs):
+    references = event.as_event.references
+    return render_template('events/management/reference_list.html', event=event, references=references, **kwargs)
```

Comment:
`references=events.references` and don't pass `**kwargs` (you only need to accept kwargs in the function to avoid breaking it when adding another kwarg there at some point)